app/src/main/java/org/simple/clinic/newentry/form/LandlineOrMobileField.kt

```diff
@@ -0,0 +1,16 @@
+package org.simple.clinic.newentry.form
+
+import org.simple.clinic.newentry.form.ValidationError.LengthTooLong
+import org.simple.clinic.newentry.form.ValidationError.LengthTooShort
+import org.simple.clinic.newentry.form.ValidationError.MissingValue
+
+class LandlineOrMobileField : InputField<String>() {
+  override fun validate(value: String): Set<ValidationError> {
+    return when {
+      value.isBlank() -> setOf(MissingValue)
+      value.length < 6 -> setOf(LengthTooShort)
```

Comment:
The numbers 6 and 12 here are very specific to a given country. We might have to change these later on to support different values based on the country.

Looks good for now, though. I think we should document these magic constants somewhere. I'll see if I can find out where these numbers came from.

Reply:
Ok, let's figure out how we'll fix this later.